### PR TITLE
MSAY hotkey and updates the controls list

### PR DESCRIPTION
**Pull request by tortellinitony**
:cl: Tortellini Tony
add: F4 hotkey for mentor chat.
/:cl:

### CODE REVIEW

**Comment by Carbonhell** [comment]
Looking at the code, f4 hotkey isn't the only thing this PR add. Can you explain why you mirrored the entire interface.dm file? And perhaps add it to the changelog too thanks

File: hippiestation.dme [hunk 1]
```diff
@@ -2773,8 +2774,8 @@
 #include "hippiestation\code\modules\tts\tts.dm"
 #include "hippiestation\code\modules\uplink\uplink_item.dm"
 #include "hippiestation\code\modules\vehicles\pimpin_ride.dm"
+#include "hippiestation\interface\interface.dm"
 #include "hippiestation\interface\skin.dmf"
-#include "interface\interface.dm"
```
Comment (praisenarsie) [inline hippiestation.dme:2777]: @Carbonhell this is why

**Comment by tortellinitony** [comment]
so we can add our own controls interface is what tells you which buttons to press

File: hippiestation.dme [hunk 2]
```diff
@@ -2773,6 +2774,7 @@
 #include "hippiestation\code\modules\tts\tts.dm"
 #include "hippiestation\code\modules\uplink\uplink_item.dm"
 #include "hippiestation\code\modules\vehicles\pimpin_ride.dm"
+#include "hippiestation\interface\interface.dm"
```
Comment (ghost) [inline hippiestation.dme:2777]: Uncheck the /tg/ interface file if we are mirroring it

Comment (tortellinitony) [reply hippiestation.dme:2777]: we just need to redefine the one proc

Comment (tortellinitony) [reply hippiestation.dme:2777]: Nevermind we will have to mirror the whole file

**Comment by Carbonhell** [comment]
yeah sure +1

**Comment by ghost** [comment]
ok +1

**Review by Carbonhell — APPROVED** [review]

**Pull request merged**